animata/feature-cards/confirmation-message.tsx

```diff
@@ -0,0 +1,100 @@
+import { motion } from "framer-motion";
+
+import { cn } from "@/lib/utils";
+
+interface ConfirmationMessageProps {
+  /**
+   * The message to appear in green box when the process will be successfully completed.
+   */
+  successMessage: string;
+
+  /**
+   * The name of the organization/bot who perform the operations.
+   */
+  labelName: string;
+
+  /**
+   * The brief about the process/text/output.
+   */
+  labelMessage: string;
+
+  /**
+   * Class name for the background element.
+   */
+  backgroundClassName?: string;
+
+  /**
+   * Class name for the container element.
+   */
+  containerClassName?: string;
+}
+
+export default function ConfirmationMessage({
+  successMessage = "Process Successful",
+  labelName = "Animata",
+  labelMessage = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque tincidunt. Vestibulum eleifend nibh a est tristique sollicitudin.",
```

Comment:
Let's add a meaningful sentence here.

Reply:
suggestion are welcome.. for this "meaningful sentence" @hari